.eslintrc.js

```diff
@@ -77,7 +80,8 @@ module.exports = {
     'no-multiple-empty-lines': 'off',
     'no-new-wrappers': 'error',
     'no-param-reassign': 'error',
-    'no-redeclare': 'error',
+    'no-redeclare': 'off',
+    '@typescript-eslint/no-redeclare': ['error'],
```

Comment:
When upgrading `@typescript-eslint` we had to update this rule: https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-redeclare.md#disallow-variable-redeclaration-no-redeclare